src/components/usercomponents/LoginForm.tsx

```diff
@@ -0,0 +1,172 @@
+            {' '}
+            <img src={Logo} alt="logo" className="h-6 mr-1" />
+            Exclusive
+          </h1>
+          <h1 className="text-center font-semibold mt-5 text-xl tracking-wider text-gray-900 Outfit">
+            Sign In
+          </h1>
+          <h5 className="text-center m-2 text-sm text-gray-800 mb-5 Outfit">
+            Enter your email to sign in for this app
+          </h5>
+          <div className="mb-4">
+            <input
+              {...register('email')}
+              id="email"
+              type="email"
+              placeholder="email@domain.com"
+              className="shadow appearance-none border rounded w-full py-1 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+            />
+            {errors.email && typeof errors.email.message === 'string' && (
+              <p className="text-red-500 text-md">{errors.email.message}</p>
+            )}
+          </div>
+          <div className="mb-4">
+            <input
+              {...register('password')}
+              id="password"
+              type="password"
+              placeholder="Password"
+              className="shadow appearance-none border rounded w-full py-1 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+            />
+            {errors.password && typeof errors.password.message === 'string' && (
+              <p className="text-red-500 text-md">{errors.password.message}</p>
+            )}
+          </div>
+          <div className="flex flex-col items-center  text-white justify-between mb-2">
+            <button
+              type="submit"
+              onClick={handleSubmit(onSubmit)}
+              className="bg-black hover:bg-gray-500  font-bold py-1 px-4 rounded focus:outline-none focus:shadow-outline bg-black-500 mb-2 w-full rounded-lg tracking-wider font-light py-2.5 text-md"
+            >
+              Sign in with email
+            </button>
+            <button className="bg-black hover:bg-gray-500  font-bold py-1 px-4 rounded focus:outline-none focus:shadow-outline bg-black-500 mb-2 w-full rounded-lg my-2 tracking-wider font-light py-2.5 text-md">
+              Forgot Password
+            </button>
+            <button className="bg-black hover:bg-gray-500  font-bold py-1 px-4 rounded focus:outline-none focus:shadow-outline bg-black-500  text-white mb-2 w-full rounded-lg my-2 tracking-wider font-light py-2.5 text-md">
+              Go Back
+            </button>
+            {isSuccess && (
+              <p className="text-green-600 text-xs font-normal">
+                Login successful!
+              </p>
+            )}
+            <h1 className="my-5  text-gray-500 ">or continue with</h1>
+            <button className="bg-black hover:bg-gray-500  font-bold py-1 px-4 rounded focus:outline-none focus:shadow-outline hover:bg-black hover:text-white w-full bg-gray-100 text-black flex flex-row text-gray-600">
```

Comment:
These buttons almost have the same classes. Can't you make this a component or use the already existing button component and extra classes needed through the className prop?